tokio/tests/rt_metrics.rs

```diff
@@ -93,7 +93,7 @@ fn try_block_threaded(rt: &Runtime) -> Result<Vec<mpsc::Sender<()>>, mpsc::RecvT
 
             // Spawn a task per runtime worker to block it.
             rt.spawn(async move {
-                tx.send(()).unwrap();
+                tx.send(()).ok();
```

Comment:
I made this fail silently because I triggered a panic here when I `^C`ed a test run, which I found needlessly confusing.